src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -334,7 +334,8 @@ function submitFormDirective(
        * Returns a callback for form submission success, which updates UI
        * state and Google Analytics.
        */
-      const handleSubmitSuccess = () => {
+      const handleSubmitSuccess = (response) => {
+        scope.responseId = response.submissionId
```

Comment:
again, let's refer to the submission ID as `submissionId` in the code for consistency